profiles/migrations/0006_enhanced_profiles.py

```diff
@@ -0,0 +1,111 @@
+# Generated by Django 2.2.10 on 2020-04-22 21:24
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import profiles.models
+
+
+class Migration(migrations.Migration):
```

Comment:
Would it be possible to add a python operation to create empty `LegalAddress` objects for each existing `Profile` in the database?